packages/desktop/src/widget_factory.rs

```diff
@@ -320,6 +345,9 @@ impl WidgetFactory {
       .get_webview_window(widget_id)
       .context("No Tauri window found for the given widget ID.")?;
 
+    #[cfg(target_os = "windows")]
+    let _ = window.as_ref().window().deallocate_app_bar();
```

Comment:
Would it work to deallocate the app bar on window destroy instead? (i.e. [here](https://github.com/glzr-io/zebar/blob/e394f9d7f5968224103798bfa258b92389134dd2/packages/desktop/src/widget_factory.rs#L301)) Cause then we could ensure the space is unreserved regardless of how the window is killed

Reply:
to get the window handle to deallocate in the WindowEvent::Destroyed event, i had to wrap the HWND in a struct that implements Send because HWND doesnt for whatever reason

```rs
struct SendHWND(windows::Win32::Foundation::HWND);
unsafe impl Send for SendHWND {}
```

i dont think this should cause issues but i dont know for sure